__TESTS__/unit/actions/Border.test.ts

```diff
@@ -5,7 +5,7 @@ import CloudinaryConfig from "../../../src/config/CloudinaryConfig";
 import * as Colors from "../../../src/constants/colors/Colors";
 import expectESMToMatchDefault from "../../TestUtils/expectESMToMatchDefault";
 
-const {solid} = Border;
+const {type, SOLID} = Border;
```

Comment:
this should be borderType no? otherwise it might collide with other names

Reply:
All these changes are aligned to the Spec,
We can open a bug about it in the spec, but this is the agreement we have at the moment (align to the spec, discuss issues later)